driver-core/src/main/java/com/datastax/driver/core/HostConnectionPool.java

```diff
@@ -464,6 +480,9 @@ else if (chosen.state.compareAndSet(TRASHED, RESURRECTING))
     }
 
     private void maybeSpawnNewConnection() {
+        if (!host.convictionPolicy.canReconnectNow())
```

Comment:
If `! canReconnectNow()`, it means connections were closed for bad reasons a short while ago.